package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "resin-typescript-skeleton",
+  "version": "0.0.1",
+  "description": "Skeleton template for a Resin TypeScript project",
+  "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+  "preferGlobal": true,
+  "types": "build/**/*.d.ts",
```

Comment:
Do wildcards work here? I hadn't really thought this through before, but looking at it now this surprises me a bit. I think `types` is supposed to point to the types of the root main module (so currently `./build/implement-me.d.ts`). As `main` maps `require('my-module')` to a specific script, `types` maps `require('my-module')` to the module's full type definition.

If you want any of the other definitions, you import them by hand as you would for a specific file in any other module (`import { GithubAction } from 'procbots/build/githubbot-api`), or your root type definition can export them explicitly (`import { GithubBot, GithubAction } from 'githubbot'`), if they're a core part of the external API.